proto/sdk/blocksdk/v1/tx.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package sdk.blocksdk.v1;
+
+option go_package = "github.com/skip-mev/block-sdk/x/blocksdk/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/msg/v1/msg.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/api/annotations.proto";
+import "amino/amino.proto";
+import "sdk/blocksdk/v1/blocksdk.proto";
+
+// Msg defines the x/blocksdk Msg service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  rpc RegisterLane(MsgRegisterLane) returns (MsgRegisterLaneResponse) {
```

Comment:
WB Deleting a Lane? If we have a `RegisterLane` flow, I'd imagine we shld also introduce a `RemoveLane` flow as well, no? However, in both cases (`Register`, `Delete`) chain devs will have to configure the lanes in the app anyway, so I'm not sure what invariant / feature these methods (`Register`, `Delete`) are intended to introduce?

Reply:
Then again, I may be misunderstanding the expected wiring logic